controller.go

```diff
@@ -94,7 +95,11 @@ func NewController(
 				StartTime:   newJob.Status.StartTime,
 			}
 			for name, n := range notifications {
-				err := n.NotifyStart(messageParam)
+				if cronJob == "" {
```

Comment:
I don't think we should do this since it breaks the previous behaviour of this tool.

How about looking at an annotation like `kube-job-notifier/notify`

If this is not present or looks like `kube-job-notifier/notify = start,success,failed` then the current behaviour is preserved, if you only wanted to notify on sucess you could set `kube-job-notify/notify = failed`

another alternative could be three annotations like `kube-job-notifier/start`, `kube-job-notifier/success` and `kube-job-notifier/failed` and these could be set to `notify` or `ignore` to override the default behaviour from the global environment variables!